start: [3,3]
model_size: [0.2, 0.2]
B: [0.1, 0.1]
frame_id: "map"
